charts/ssi-asr/values.yaml

```
@@ -21,10 +21,10 @@
 authorities:
   # -- The first authority
   authorityOne:
-    bpn: "BPNL00000003CRHK"
+    did: "did:web:portal-backend.int.catena-x.net:api:administration:staticdata:did:BPNL00000003CRHK"
```

Please don't use any real but only dummy values:
```suggestion
    did: "did:web:example.org:api:administration:staticdata:did:BPNL00000003CRHK"
```

This occurs 13 times in your change, please fix all occurrences.